viz_scripts/generic_timeseries.ipynb

```diff
@@ -168,11 +168,6 @@
     "            emissions['Mode_confirm_kg_CO2'] = emissions['Mode_confirm_kg_CO2'].fillna(0)\n",
     "            emissions['distance'] = emissions['Mode_confirm_kg_CO2'].fillna(0)\n",
     "\n",
-    "            # Sum daily energy for each user\n",
```

Comment:
Reverted the changes to keep the code block, since I have made a single function for both default and dynamic mapping.
For the dynamic config changes, we're not displaying the energy calculations on the public dashboard.

Reply:
I'm not going to hold up the merge for this change, but now that you have added an `if` block, you may want to clarify why there is no `else` as a code comment.